bootstrap/pkg/utils/iamutils_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2018 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"fmt"
+	"testing"
+
+	"google.golang.org/api/cloudresourcemanager/v1"
+)
+
+func Test(t *testing.T) {
+	tests := []struct {
```

Comment:
Thanks. So the typical style in go is to define a list of testCases where each testCase would specify the input and the expected output.

Would it be possible to write the test that way?

e.g. add a struct variable that should equal the expectedPolicy returned by GetClearediamPolicy. Then do a deepEqual to compare the expected and actual. 

Something like:
https://github.com/kubeflow/tf-operator/blob/cf01b5125e25239042e1caef1ef44f7aaf2337e3/pkg/apis/tensorflow/v1beta1/defaults_test.go#L117

Reply:
updated